browser/profiles/brave_bookmark_model_loaded_observer.cc

```diff
@@ -5,8 +5,11 @@
 
 #include "brave/browser/profiles/brave_bookmark_model_loaded_observer.h"
 
+#include "brave/common/pref_names.h"
```

Comment:
do any deps need to be added for these?

Reply:
fixed in 07e019e5ed6b0dc01072bcc11f5b9bfa60ef9150